src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -291,7 +297,7 @@ def test_req_no_inputs__fail(self):
         expected = {
             STATUS_BATCH: JOB_NOT_PROVIDED_ERR,
             RETRY: JOBS_NOT_PROVIDED_ERR,
-            LOGS: JOB_NOT_PROVIDED_ERR,
+            LOGS: JOBS_NOT_PROVIDED_ERR,
```

Comment:
takes `job_id_list` now instead of `job_id` => error message is the plural version